src/Features/Core/Portable/UnifiedSuggestions/UnifiedSuggestedActions/UnifiedCodeRefactoringSuggestedAction.cs

```diff
@@ -0,0 +1,30 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+#nullable enable
+
+using Microsoft.CodeAnalysis.CodeActions;
+using Microsoft.CodeAnalysis.CodeRefactorings;
+
+namespace Microsoft.CodeAnalysis.UnifiedSuggestions
+{
+    /// <summary>
+    /// Similar to CodeRefactoringSuggestedAction, but in a location that can be used by
```

Comment:
Same here `CodeRefactoringSuggestedAction` is an internal type, and it would be unfortunate to have two separate internal implementations.

Reply:
Can we add an `ICodeRefactoringSuggestedAction` that is implemented by both `CodeRefactoringSuggestedAction` and `UnifiedCodeRefactoringSuggestedAction`?